code/go/0chain.net/smartcontract/storagesc/models.go

```diff
@@ -1356,45 +1356,33 @@ func (sa *StorageAllocation) replaceBlobber(blobberID string, sc *StorageSmartCo
 	for i, d := range sa.BlobberAllocs {
 		if d.BlobberID == blobberID {
 			blobberIsKilled := false
-			var getBlobberError error
-			actErr := cstate.WithActivation(balances, "apollo", func() (e error) { return },
-				func() (e error) {
-					blobber, e := sc.getBlobber(d.BlobberID, balances)
-					if e != nil {
-						getBlobberError = e
-						return
-					}
-
-					bb := blobber.mustBase()
+			blobber, e := sc.getBlobber(d.BlobberID, balances)
+			if e != nil {
+				return common.NewError("remove_blobber_failed",
+					"can't get blobber "+d.BlobberID+": "+err.Error())
```

Comment:
We return the error directly before actually, see how the actErr was handled, the `getBlobberError` is checked after the `actErr` is checked.

Reply:
Fixed.